### PR TITLE
RUB-17 Move aliasing logic inside core gem

### PR DESCRIPTION
**Pull request by ashish91**

### CODE REVIEW

**Comment by ashish91** [comment]
@rtdp Please revierw.

File: lib/generators/language_alias_generator.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  rescue StandardError => e
+    puts e.message
+  end
+
+  def alias_instance_method(class_name, orig_name, alias_name)
+    klass = Object.class_eval(class_name)
+
+    klass.send(:alias_method, alias_name, orig_name)
+  rescue StandardError => e
+    puts e.message
```
Comment (rtdp) [inline lib/generators/language_alias_generator.rb:17]: Do we benefit from capturing the error and simply putting the message instead?

File: lib/templates/lib/rubyvernac/language_alias_loader.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      # Object.class_eval(" #{class_trans} = #{class_name} ") unless class_trans.length.zero?
+
+      # class methods -
+      content.first.last['cpumethods'].each do |k, v|
+        #puts "syncing -- #{k} to #{v}"
+        begin
+          @parser.alias_class_method(class_name, k, v) unless v.chop.length.zero?
+        rescue StandardError => e
+
+        end
```
Comment (rtdp) [inline lib/templates/lib/rubyvernac/language_alias_loader.rb:35]: This has potential for creating silent failures and hard to debug issues. Do we need to handle standard error here?

File: lib/generators/language_alias_generator.rb (hunk 1 above)
Comment (ashish91) [reply lib/generators/language_alias_generator.rb:17]: I am not aware of what kind of error it throws which was meant to be captured by StandardError. The original code expects an error.
Basically having Custom error classes helps in debugging in logs. In this case I didn't knew what type of error we're expecting so I added puts.
We can skip rescuing the error as well, although the error message will not be obvious. Do you want me to remove it ?

File: lib/templates/lib/rubyvernac/language_alias_loader.rb (hunk 2 above)
Comment (ashish91) [reply lib/templates/lib/rubyvernac/language_alias_loader.rb:35]: I'll remove the rescue block. Thanks

File: lib/templates/lib/rubyvernac/language_alias_loader.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
+require 'yaml'
+
+class LanguageAliasLoader
```
Comment (rtdp) [inline lib/templates/lib/rubyvernac/language_alias_loader.rb:3]: Is this class going inside the language gem or staying core? Preference would be to keep this in core-gem.

Comment (ashish91) [reply lib/templates/lib/rubyvernac/language_alias_loader.rb:3]: I've moved this logic into core gem.

**Pull request merged**